### PR TITLE
depens:expact update

### PR DESCRIPTION
**Pull request by ArqTras**

### CODE REVIEW

**Comment by ArqTras** [comment]
@selsta messed with last commit so this shoudl be ok

File: contrib/depends/packages/expat.mk [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package=expat
 $(package)_version=2.2.4
-$(package)_download_path=https://downloads.sourceforge.net/project/expat/expat/$($(package)_version)
+$(package)_download_path=https://github.com/libexpat/libexpat/releases/download/$($(package)_version)
```
Comment (selsta) [inline contrib/depends/packages/expat.mk:3]:
```suggestion
$(package)_download_path=https://github.com/libexpat/libexpat/releases/download/R_2_2_4
```

**Comment by selsta** [comment]
please squash again :)

**Comment by ArqTras** [comment]
sorry messed with my local repo

**Comment by ArqTras** [comment]
#6500

**Pull request closed without merging**